modules/util/portal-tools-rest-builder/src/main/java/com/liferay/portal/tools/rest/builder/RESTBuilder.java

```diff
@@ -232,7 +237,9 @@ public void build() throws Exception {
 
 				Schema schema = entry.getValue();
 
-				_putSchema(context, schema, schemaName);
+				_putSchema(
+					context, schema, schemaName,
+					_getSubentities(allSchemas, javaMethodSignatures));
```

Comment:
But subschemas can be exported to the context like allSchemas, but I guess they are scoped to a schema so it doesn't make sense